sdk/log/record.go

```diff
@@ -37,6 +37,7 @@ type Record struct {
 	// Allocation optimization: an inline array sized to hold
 	// the majority of log calls (based on examination of open-source
 	// code). It holds the start of the list of attributes.
+	// TODO: Not sure whether to use a normal array here or not, as we know the size when copy the attributes from log.Record.
```

Comment:
@open-telemetry/go-approvers, any options about this?

Should we use a normal array here? Like `attributes []log.KeyValue`

Reply:
You can check the prototype for implementation.

Reply:
> @open-telemetry/go-approvers, any options about this?
> 
> Should we use a normal array here? Like `attributes []log.KeyValue`

I think that's worth exploring, but I would do so in a follow up PR.